### PR TITLE
Add eBPF for Windows as an emerging project

### PR DESCRIPTION
**Pull request by dthaler**
No logo exists yet, but a logo is not one of the
"Requirements for a project to be listed" at
https://ebpf.io/projects/

Signed-off-by: Dave Thaler <dthaler@microsoft.com>

### CODE REVIEW

File: src/pages/projects.js [hunk 1]
```diff
@@ -422,6 +422,35 @@ const ProjectDescriptions = () => (
 
     <TitleWithAnchor headerClassName="projects-title projects-common-title">Emerging Projects</TitleWithAnchor>
 
+    <div className="project-box">
+      <TitleWithAnchor headerClassName="project-major-title projects-common-title" headerTag="h3">eBPF for Windows</TitleWithAnchor>
+      <div className="project-minor-title">
+        eBPF Runtime for Windows
+      </div>
+      <div className="project-body">
+        <!-- no logo exists yet -->
```
Comment (qmonnet) [inline src/pages/projects.js:431]: Netlify complains about this comment. I'm not very familiar with the framework in use, but my guess is that even if we use some HTML tags the current file is JavaScript, and comments would be `//` or `/* */`? No idea how to _generate_ HTML comments (but then we probably just want a comment in the file anyway, not in the generated site).

File: src/pages/projects.js [hunk 2]
```diff
@@ -422,6 +422,35 @@ const ProjectDescriptions = () => (
...
+        </a>
```
Comment (qmonnet) [inline src/pages/projects.js:432]: This closing tag seems to be a leftover, I see no matching opening tag.

**Review by qmonnet — CHANGES_REQUESTED** [review]

File: src/pages/projects.js (hunk 1 above)
Comment (dthaler) [reply src/pages/projects.js]: Removed comment for now

File: src/pages/projects.js [hunk 3]
```diff
@@ -422,6 +422,33 @@ const ProjectDescriptions = () => (
...
+        eBPF Runtime for Windows
+      </div>
+      <div className="project-body">
+        <div className="project-description">
+          <p>
+            <a href="https://microsoft.github.io/ebpf-for-windows/">
+              <b>Website</b>
+            </a>{" "}
+            |{" "}
+            <a href="href="https://github.com/microsoft/ebpf-for-windows">
```
Comment (qmonnet) [inline src/pages/projects.js:437]:
```suggestion
            <a href="https://github.com/microsoft/ebpf-for-windows">
```

**Review by qmonnet — CHANGES_REQUESTED** [review]

**Review by qmonnet — APPROVED** [review]
Thanks! The PR looks good to me.
Here's what the preview looks like without a logo:

Screenshot_20210927_195304

(As per the PR linked above I understand the logo will be added as soon as available.)

**Comment by borkmann** [comment]
@dthaler I would suggest we add a new section `Emerging Infrastructure` where we can add the `eBPF for Windows` and later on I'm inclined to make a PR to also add `gcc` with eBPF backend there.

Would it be possible to make it similar to Linux section where we would have: `Windows` and `eBPF Runtime` as a subtitle? Also, would it be possible for now to just add the Windows logo there until you have a eBPF + Windows specific one?

Thanks!

**Comment by dthaler** [comment]
> @dthaler I would suggest we add a new section `Emerging Infrastructure` where we can add the `eBPF for Windows` and later on I'm inclined to make a PR to also add `gcc` with eBPF backend there.

If we did that, I'd suggest we rename "Emerging Projects" since Infrastructure projects are still projects.   (I'd also suggest we decouple the section naming issue from the addition of specific projects.)

> Would it be possible to make it similar to Linux section where we would have: `Windows` and `eBPF Runtime` as a subtitle so that it is consistent for the different OSes? 

Currently there is no Linux section.   There is a "Core Infrastructure" section but it is not Linux specific (the LLVM compiler project for example is platform agnostic, and we use it as is for Windows). 

> Also, could we just add the Windows logo there until you have a eBPF + Windows specific one?

FYI, having a logo is not listed as a requirement to be listed, if we want to add that as a requirement I think it would require a PR to update the requirements.

As for your request, I would prefer not to, since it is not Windows itself and it would require legal approval to use the Windows logo and given that's not what we want anyway, going through the process for something we don't want to use seems burdensome.

**Comment by dthaler** [comment]
Updated subtitle to say just "eBPF Runtime" per @borkmann 
I did not change the title to be "Windows" since unlike Linux, it is not part of Windows itself.  The name of the project should be the title, and the name of the project is "eBPF for Windows".

**Review by lmb — APPROVED** [review]

**Review by anakryiko — APPROVED** [review]

**Comment by dthaler** [comment]
Rebased on top of the "Core Infrastructure (Emerging)" section addition, where this now appears

**Review by qmonnet — CHANGES_REQUESTED** [review]
The link to the Slack channel seems broken, it redirects to a channel but does not specify a Slack space.

**Pull request merged**